src/style.css

```diff
@@ -0,0 +1,110 @@
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: "Inter", sans-serif;
+  font-size: 1.6rem;
+  color: #333;
+}
+
+.header,
+.footer {
+  min-height: 6rem;
+  background-color: #c1c1c1;
+  display: flex;
+  align-items: center;
+}
+
+.main {
+  min-height: calc(100vh - 12rem);
+  padding: 4rem 0;
+}
+
+.row {
+  width: 80%;
+  margin: 0 auto;
+}
+
+.board-header {
+  padding-bottom: 4rem;
+}
+
+.board-content {
+  display: flex;
+  gap: 4rem;
+}
+
+.list-header {
+  display: flex;
+  align-items: center;
+  gap: 6rem;
+}
```

Comment:
- Please let's kindly improve our design on this part to match the wireframe as much as we can. see the images below:


*wireframe*
![Screenshot from 2022-11-16 00-36-03](https://user-images.githubusercontent.com/59397159/202047766-6c7057e9-50ab-44ab-860d-f7314d0dd264.png)

*your design*
![Screenshot from 2022-11-16 00-34-47](https://user-images.githubusercontent.com/59397159/202047773-589d15a0-47e6-43a1-a2cd-50da71a06a4c.png)